config/jetpack-cloud-development.json

```diff
@@ -51,7 +51,8 @@
 		"oauth": true,
 		"realtime-site-count": true,
 		"site-indicator": false,
-		"support-user": true
+		"support-user": true,
+		"upgrades/redirect-payments": true
```

Comment:
Feature flag must be enabled to enable Paypal and have at least one payment method. See https://github.com/Automattic/wp-calypso/blob/61acc957638fc7009632af4c0c058aa6bc0e82b7/client/my-sites/checkout/composite-checkout/lib/is-payment-method-enabled.ts